cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -71,7 +80,7 @@ describe('Topology view', () => {
       });
   });
 
-  it('should show adjust instance forks when instance forks slider is adjusted', () => {
+  it.skip('should show adjust instance forks when instance forks slider is adjusted', () => {
```

Comment:
I see that you have skipped this test- It looks like it would be a better component test. Perhaps it can be moved to the component test file for this feature, and deleted from this file?